scripts/module/formapplications/sequencer-database-viewer.js

```diff
@@ -1,179 +1,228 @@
-export default class SequencerDatabaseViewer extends FormApplication {
-
-    constructor(dialogData = {}, options = {}) {
-        super(dialogData, options);
-        this.filter = "all"
-        this.search = "";
-        this.autoplay = true;
-        this.packs = Object.keys(window.Sequencer.Database.entries);
-        this.entries = window.Sequencer.Database.flattenedEntries.map(entry => {
-            return {
-                pack: entry.split('.')[0],
-                entry: entry
-            };
-        });
-        this.list = false;
-    }
-
-    static show(inFocus = false){
-        if (!game.user.isTrusted) return;
-        for(let app of Object.values(ui.windows)){
-            if(app instanceof this){
-                return app.render(true, { focus: inFocus });
-            }
-        }
-        return new this().render(true);
-    }
+import { cache } from "../lib/cache.js";
+import { reactiveEl as html } from "../lib/html.js";

```

Comment:
Guide: by importing this as `html` we can trick github and vscode (with the lithtml extension) to allow us to get syntax highlighting for the html we write when we use the function